docs/grpc_api.md

```diff
@@ -0,0 +1,70 @@
+# TorchServe gRPC API
+
+TorchServe also supports [gRPC APIs](../frontend/server/src/main/resources/proto) for both inference and management calls.
+
+TorchServe provides following gRPCs apis
+
+* [Inference API](../frontend/server/src/main/resources/proto/management.proto)
```

Comment:
"../frontend/server/src/main/resources/proto/management.proto" should be changed to "../frontend/server/src/main/resources/proto/inference.proto"
```suggestion
* [Inference API](../frontend/server/src/main/resources/proto/inference.proto)
```